src/offset/local/tz_info/timezone.rs

```diff
@@ -817,14 +817,16 @@ mod tests {
                 let time_zone_local_1 = TimeZone::from_posix_tz(&tz)?;
                 assert_eq!(time_zone_local, time_zone_local_1);
             } else {
-                let time_zone_local = TimeZone::local()?;
-                let time_zone_local_1 = TimeZone::from_posix_tz("localtime")?;
-                let time_zone_local_2 = TimeZone::from_posix_tz("/etc/localtime")?;
-                let time_zone_local_3 = TimeZone::from_posix_tz(":/etc/localtime")?;
-
-                assert_eq!(time_zone_local, time_zone_local_1);
-                assert_eq!(time_zone_local, time_zone_local_2);
-                assert_eq!(time_zone_local, time_zone_local_3);
+                // To be discussed - does this make sense in the context
```

Comment:
What should we do with these tests now?

Reply:
I think removing is the best option, they are just re-implementing the logic of `TimeZone::local()` anyway, and to make it work we'd also have to re-implement the UTC fallback here. The first case with TZ env var is useful as that verifies that the env var overrides other settings

Reply:
Sounds good.